src/modules/frequency/application/frequency.service.query.interfaces.ts

```diff
@@ -1,6 +1,15 @@
-import { UserClassesDTO } from "./frequency.dtos";
+import {
+  EnrolledStudentDTO,
+  StudentClassAttendanceItemDTO,
+  StudentGeneralAttendanceResponseDTO,
+  UserClassesDTO } from "./dtos";
 
 export const FREQUENCY_QUERIES_TOKEN = "IFrequencyQueries";
-export interface IFrequencyQueries{
-    getMyClasses(userId: number): Promise<UserClassesDTO[]>;
-}
\ No newline at end of file
+export interface IFrequencyQueries {
+  getMyClasses(userId: number): Promise<UserClassesDTO[]>;
```

Comment:
This userId is temporary, right? We'll extract it from req headers?
```suggestion
  getMyClasses(): Promise<UserClassesDTO[]>;
```

Reply:
Isso. Essa informação virá do header da requisição HTTP. Será mantido por enquanto. Futuramente, isso deverá ser alterado.